dockers/docker-fpm-frr/zebra.interfaces.conf.j2

```diff
@@ -39,13 +39,23 @@ link-detect
 {%     endif %}
 {%   endfor %}
 {% endif %}
+{% if lo_ipv4_addrs|length > 0 or lo_ipv6_addrs|length > 0 %}
+interface Loopback0
+{% if lo_ipv4_addrs|length > 0 %}
+    ip address {{ lo_ipv4_addrs[0] }}
+{% endif %}
+{% if lo_ipv6_addrs|length > 0 %}
+    ipv6 address {{ lo_ipv6_addrs[0] | ip }}/{{ lo_ipv6_addrs[0] | prefixlen }}
```

Comment:
this does not subscript the config db lo address change.

Reply:
Yes, so does all configs in templates, there are several commands referenced the lo address. To follow changes, a lot of work to do. I think it is a new requirement.